vscode/interop/interop.mli

```diff
@@ -18,6 +18,10 @@ module Regexp : sig
   val t_to_js : Js_of_ocaml.Regexp.regexp -> Ojs.t
 
   val t_of_js : Ojs.t -> Js_of_ocaml.Regexp.regexp
+
+  type replacer = string -> string list -> string
+
+  val replace : string -> regexp:t -> replacer:replacer -> string
```

Comment:
Surprised that this binding doesn't exist in jsoo. You should consider contributing it.

Reply:
This function is difficult to type correctly. It includes a variable number of string parameters with different parameter types after that: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace#Specifying_a_function_as_a_parameter

Now that I've revisited it, I think this binding is unsafe due to the trailing parameters that were omitted.

I'll try to contribute it to Js_of_ocaml if I can get a gen_js_api version working first.